.github/workflows/ci.yml

```diff
@@ -38,15 +38,14 @@ jobs:
     name: Python compatibility test
     needs: [ pre-commit, towncrier, package ]
     runs-on: ubuntu-latest
-    continue-on-error: ${{ matrix.experimental }}
+    continue-on-error: ${{ matrix.experimental || false }}
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12", "3.13-dev" ]
+        python-version: [ "3.9", "3.10", "3.11", "3.12", "3.13", "3.14" ]
         include:
-          - experimental: false
           # Development Python can fail without failing the entire job
-          - python-version: "3.13-dev"
+          - python-version: "3.14"
```

Comment:
I think it's clearer to keep the default `experimental` value in the `matrix` section, because:
* It allows you to derive all the values from that section alone.
* It avoids needing to repeat the default value if it was used in more than one place.